### ISSUE TITLE
Reproduce results

### ISSUE THREAD

**Issue by liangshenglei** [namepllet/HandOccNet#35]
Great job, currently I am reproducing your code. In the ho3d dataset, like most people, I did not reach the results in the paper, possibly due to training strategy issues. 
![image](https://github.com/user-attachments/assets/62849c3d-e777-41cb-9a31-900be7323c4a)
However, when reproducing the DexYCB dataset, I found that the results were quite different and the model did not seem to fully converge. This is my 25th epoch loss. The environment I replicated was 4 blocks of 3090, and the learning rate did not change. 
![image](https://github.com/user-attachments/assets/fcfba850-9f8b-4c14-a35b-625a89f283fe)

I would like to ask how to achieve the approximate results in the paper. I hope to receive your help, thank you very much!!!

**Comment by M-leng** [comment]
Hi, How did you solve this problem?
